src/ProjectNeon/Assets/Features/Combatants/MemberState.cs

```diff
@@ -46,6 +50,14 @@ public MemberState(IStats baseStats)
     public void GainPrimaryResource(int numToGive) => _counters[PrimaryResource.Name].ChangeBy(numToGive);
     private IResourceType PrimaryResource => ResourceTypes[0];
 
+    public void FeedOn(string feedType)
+    {
+        if (_counters.ContainsKey(feedType))
+        {
+            _counters[feedType] = new BattleCounter((TemporalStatType)Enum.Parse(typeof(TemporalStatType), feedType), 1, () => 1);
```

Comment:
Code in this class needs to be clean. This is one of the most crucial classes in the whole game. 

If this should be an enum, it should come in as an enum in the argument.

Reply:
@SilasReinagel  Solved